react/components/PickupDetailsModal.tsx

```diff
@@ -0,0 +1,148 @@
+  pickupOption,
+  showPickupModal,
+  setShowPickupModal,
+}) => {
+  return (
+    <Modal
+      isOpen={showPickupModal}
+      onClose={() => setShowPickupModal(false)}
+      title={<FormattedMessage {...messages.details} />}
+    >
+      <div className="lh-copy">
+        <Divider orientation="horizontal" />
+        <div className="flex-column mt4 mb4">
+          <div className="flex justify-between mb0">
+            <div className="fw6 f3">{pickupOption.friendlyName}</div>
+            <div className="fw5 f5">
+              <FormattedPrice value={(pickupOption.price ?? 0) / 100} />
+            </div>
+          </div>
+
+          <div className="fw5 f5 c-muted-1">
+            <TranslateEstimate shippingEstimate={pickupOption.estimate ?? ''} />
+          </div>
+
+          <div className="fw4 f6 c-muted-1">
+            <FormattedMessage
+              id="store/checkout.shipping.distance"
+              values={{
+                distanceValue: pickupOption.storeDistance.toFixed(1),
+              }}
+            />
+          </div>
+        </div>
+
+        <div className="flex-column mb4">
+          <span className="fw6 mb2">
+            <FormattedMessage id="store/checkout.shipping.pickupPointsModal.address" />
+          </span>
+          <div>{`${pickupOption.address?.street}, ${pickupOption.address?.number}`}</div>
+          <div>{`${pickupOption.address?.neighborhood} - ${pickupOption.address?.city} - ${pickupOption.address?.state}`}</div>
+        </div>
+
+        <div className="flex-column mb4">
+          <span className="fw6 mb2">
+            <FormattedMessage id="store/checkout.shipping.pickupPointsModal.businessHours" />
+          </span>
+          {pickupOption.businessHours.map((businessHour: BusinessHour) => (
+            <>
+              <div className="flex justify-between mb2 mt2">
+                <FormattedMessage
+                  id={`store/checkout.shipping.pickupPointsModal.weekDay${businessHour.dayNumber}`}
+                />
+                <div>
+                  {businessHour.closed ? (
+                    <FormattedMessage id="store/checkout.shipping.pickupPointsModal.closed" />
+                  ) : (
+                    <div>
+                      <FormattedMessage
+                        id="store/checkout.shipping.pickupPointsModal.hourFromTo"
+                        values={{
+                          openingTime: formatHour(businessHour.openingTime),
+                          closingTime: formatHour(businessHour.closingTime),
+                        }}
+                      />
+                    </div>
+                  )}
+                </div>
+              </div>
+              <Divider orientation="horizontal" />
+            </>
+          ))}
+        </div>
+
+        <div className="flex-column">
+          <div className="fw6 mb2">
+            <FormattedMessage id="store/checkout.shipping.pickupPointsModal.availableItems" />
+          </div>
```

Comment:
same as the comment above about the semantic elements, whichever you chose to go with (heading or spans) you need to change here as well

Reply:
Thanks! Done.